.copr/Makefile

```diff
@@ -0,0 +1,15 @@
+pkg_install := $(shell dnf -y install git rpm-build)
```

Comment:
I guess this is associated with this build: https://copr.fedorainfracloud.org/coprs/cleber/avocado-copr-test/build/856472/ so it looks like it works well...